src/components/Subjects/Datalist.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { PRIMARY_SUBJECTS } from '../../constants/SubjectTypes'
+
+const Datalist = ({ subjectsType }) => {
+  if (subjectsType === PRIMARY_SUBJECTS) {
+    return (
+      <datalist id={subjectsType}>
+        <option value='Biologia' />
```

Comment:
You could store values like this
```
const dataOptions = {
  primary: ['Biologia', 'Chemia', etc],
  lingual: ['Język angielski', 'Język niemiecki', etc],
}
```
And generate the markup from iterating over them.

However this solution is fine, so it's up to you which one you go for.